docs/docs/components/api/README.md

```diff
@@ -736,6 +736,24 @@ async run({ $ }) {
 
 It functions the same way as [`$end` in workflow code steps](/workflows/steps/code/#end).
 
+**`$.summary`**
+
+`$.summary` is used to surface brief, user-friendly summaries about what happened when an action step succeeds. For example, when [adding items to a Spotify playlist](https://github.com/PipedreamHQ/pipedream/blob/master/components/spotify/actions/add-items-to-playlist/add-items-to-playlist.mjs#L51):
+<div>
+<img alt="Spotify example with $summary" src="./images/spotify-$summary-example.png">
+</div>
+
+Example implementation:
+```
+// eslint-disable-next-line multiline-ternary
```

Comment:
I'd recommend removing the `// eslint-disable-next-line multiline-ternary` line to simplify the example